library/core/src/num/int_macros.rs

```diff
@@ -1772,9 +1772,9 @@ macro_rules! int_impl {
         #[inline]
         #[rustc_inherit_overflow_checks]
         pub const fn abs(self) -> Self {
-            // Note that the #[inline] above means that the overflow
-            // semantics of the subtraction depend on the crate we're being
-            // inlined into.
+            // Note that the #[rustc_inherit_overflow_checks] above means that
```

Comment:
If my understanding is correct, the above `#[inline]` should also have meaning here, shouldn't it? Then "#[inline] and #[rustc_inherit_overflow_checks] above mean..." would be better IMO.